examples/basic/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "@electron-webpack-examples/basic",
```

Comment:
Nitpick, but should this be `@electron-redux-webpack-examples/basic`?

Reply:
Oh, actually I didn't want webpack part here at all, must have included it by mistake, since it was giving me hard time :D It should be `@electron-redux-examples/basic`